talk/introduction/goals.tex

```diff
@@ -13,7 +13,7 @@
   \pause
   \begin{block}{Fast}
     \begin{itemize}
-    \item compiled (unlike Java, C\#, Python, ...)
+    \item compiled to native machine code (unlike Java, C\#, Python, ...)
```

Comment:
This change screws the layout of the slide (line is too long now). I've tested with
```suggestion
    \item compiled to native machine code
      \begin{itemize}
      \item unlike Java, C\#, Python, ...
      \end{itemize}
```
and this looks ok.